coursebuilder/modules/explorer/templates/explorer.html

```diff
@@ -52,17 +52,31 @@
     <div class="footer-bottom">
         <div class="row">
             <div class="col l8 m12 footer-appstores">
+                <div class="col s2">
                 <a target="_blank" href="https://codeforafrica.org/">
```

Comment:
You may replace these three logos with that in https://academy.codeforafrica.org/img/logos/powered-by-gray.png It does not very good on mobile as it is now. For consistency please refer to the footer in https://academy.codeforafrica.org/